tsfc/parameters.py

```diff
@@ -21,6 +21,7 @@
 
     # Precision of float printing (number of digits)
     "precision": numpy.finfo(NUMPY_TYPE).precision,
+    "optimisation": "O2"
```

Comment:
Let us not make the same mistake that occurred in coffee and have multiple different meanings of the optimisation levels (and places of setting them).  What is the right way to specify the optimisation level?  If I look at compilers, `O1`, `O2`, etc... are shorthand for turning on individual compiler passes.  I think coffee has some idea of this, should we do something like that here?

In particular, there's no indication here how I turn this stuff off.  What do I set optimisation to?